backend/api/views.py

```diff
@@ -600,11 +604,11 @@ def history(self, request, league_id, pk=None):
         # only add ranked, non-tournament scrimmages
         # add entry to result array defining whether or not this team won and time of scrimmage
         for scrimmage in scrimmages:
-            if (scrimmage.ranked and scrimmage.tournament_id == -1):
+            if (scrimmage.ranked):
```

Comment:
can u add back the tour_id==-1 check?
(basically, tournament messages shouldn't show up in the list, bc then that creates spoilers. our tour running isn't the best)

Reply:
oops this was mistake